multihost_test/Sanity/test_pam_pwquality.py

```diff
@@ -0,0 +1,147 @@
+                                                            "cat /tmp/anuj").stdout_text
+        execute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC k4ddL32rfg NOPASS")
+        execute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=2 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC k4ddL32rfg NOPASS")
+        execute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 difok=3 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC aR3dh4T1nCb aR3dh4T1nCb")
+        execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                               "local_anuj R3dh4T1nC aR3dh4T1nCbD aR3dh4T1nCbD")
+        execute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 minlen=5 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                               "local_anuj aR3dh4T1nCbD jf@#FafR3dh4T1nC!!F jf@#FafR3dh4T1nC!!F")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj jf@#FafR3dh4T1nC!!F 3214 3214")
+
+    def test_pam_gecoscheck(self, multihost, create_localusers):
+        """
+        :title: Sanity tests for pam_pwquality.so gecoscheck
+        :id: 60afc548-f063-11eb-9639-845cf3eff344
+        """
+        # Test gecoscheck parameter
+        execute_cmd(multihost, "echo pamtest1 | passwd --stdin pamtest1")
+        execute_cmd(multihost, "usermod -c 'Karci' pamtest1")
+        execute_cmd(multihost, "pinky -l pamtest1")
+        execute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password    "
+                               "required pam_pwquality.so "
+                               "gecoscheck=1 enforce_for_root/' "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "sh /tmp/pam_pwquality.sh  pamtest1 pamtest1 KarciMir4 KarciMir4")
```

Comment:
This line probably got backspaced by mistake, may fail pep8 check

Reply:
corrected .